Src/Support/GoogleApis.Core/Apis/Util/Utilities.cs

```diff
@@ -69,15 +69,6 @@ internal static bool IsNullOrEmpty<T>(this IEnumerable<T> coll)
             return coll == null || coll.Count() == 0;
         }
 
-        /// <summary>

```

Comment:
This is a breaking change - it's a public method.
Annoying, I know, but we should keep it.